pkgs/applications/networking/freefilesync/default.nix

```diff
@@ -0,0 +1,102 @@
+{ lib
+, gcc12Stdenv
+, fetchFromGitHub
+, fetchpatch
+, pkg-config
+, curl
+, glib
+, gtk3
+, libssh2
+, openssl
+, wxGTK32
+}:
+
+gcc12Stdenv.mkDerivation rec {
+  pname = "freefilesync";
+  version = "11.25";
+
+  src = fetchFromGitHub {
+    owner = "hkneptune";
+    repo = "FreeFileSync";
+    rev = "v${version}";
+    sha256 = "sha256-JV9qwBiF9kl+wc9+7lUufQVu6uiMQ6vojntxduNJ8MI=";
+  };
+
+  patches = [
+    # Patches from ROSA Linux
```

Comment:
Why do we need those patches?

Reply:
FreeFileSync in fact builds against [GTK 3](https://freefilesync.org/forum/viewtopic.php?t=676), and a [runtime error](https://freefilesync.org/forum/viewtopic.php?t=9444) also needs to be fixed.

Reply:
Can you add two short comments with that?

Also those issues are old...

Reply:
Done.